packages/react/src/components/Authenticator/shared/SignInSignUpTabs.tsx

```diff
@@ -20,11 +24,15 @@ export const SignInSignUpTabs = (): JSX.Element => {
            {route === 'signIn' && <SignIn />}
         </View>
       </TabItem>
-      <TabItem title={translate('Create Account')}>
-        <View data-amplify-router-content="">
-           {route === 'signUp' && <SignUp />}
-        </View>
-      </TabItem>
+      {!hideSignUp ? (
+        <TabItem title={translate('Create Account')}>
+          <View data-amplify-router-content="">
+             {route === 'signUp' && <SignUp />}
+          </View>
+        </TabItem>
+      ) : (
+        <></>
+      )}
```

Comment:
Q more on UX perspective:

With this change, authenticator behaves like


https://user-images.githubusercontent.com/43682783/152621713-b029058c-626c-4826-b2e6-1ea436040695.mp4



Top tab still looks/acts clickable because hovering on it shows `pointer` cursor and clicking on it shows font color changes. Instead of showing `Sign In` tab, should we just remove as a whole? That will resemble our previous css suggestion in https://ui.docs.amplify.aws/components/authenticator#additional-css-styling.

Reply:
[4e2bebf](https://github.com/aws-amplify/amplify-ui/pull/1260/commits/4e2bebfa56dd03bcb0f7ca2f63267c69c1fefde8)